### ISSUE TITLE
Scoring algorithm change 

### ISSUE THREAD

**Issue by psifertex** [ctftime/ctftime.org#6]
placeholder to start a discussion about the scoring algorithm and whether other approaches might work better. Could be implemented in a "beta" mode initially to tested. Heck, with the API, someone else could (should) try a few alternate algorithms out and post the results here so they could be discussed.

**Comment by kyprizel** [comment]
There were ideas to add ELO rating, not sure if it's applicable here.

**Comment by psifertex** [comment]
Yup, ELO and TrueSkill were two suggestions I've heard and am interested in seeing. TrueSkill is encumbered with patents (and trademark), but last time I checked it out, it looks like you can use the algorithm for non-profit uses as long as you call it something else.

**Comment by kyprizel** [comment]
We need to take some decision
